### DISCUSSION TITLE
An option to display simple output to terminal and save -v to a file.

### DISCUSSION THREAD

**Discussion by gprime31** [projectdiscovery/interactsh#180] [Q&A] [answered]
Like the default:
Recieved HTTP interactionfrom ip at time.    will show in the terminal.
and in the -o logs will show with the -v flag (all the info). in the save file.

**Comment by ehsandeep** [comment, marked as answer]
@gprime31 we suggest using `json` switch and processing the output as per need, `o` flag simply stores the CLI output, introducing custom behavior may cause more issues as someone will differ what needs to be saved in output vs default CLI output.

**Comment by gprime31** [comment]
Ok, thanks
